Source/Core/UICommon/GameFile.cpp

```diff
@@ -39,6 +41,16 @@ namespace UICommon
 {
 static const std::string EMPTY_STRING;
 
+bool operator==(const GameBanner& lhs, const GameBanner& rhs)
+{
+  return lhs.buffer == rhs.buffer && lhs.width == rhs.width && lhs.height == rhs.height;
```

Comment:
```cpp
return std::tie(lhs.buffer, lhs.width, lhs.height) == std::tie(rhs.buffer, rhs.width, rhs.height);
```

Reply:
Any particular reason? To me it just seems like it makes the line longer and requires an extra include.

Reply:
It's a future-proofing thing. It's less error-prone compared to chaining the comparisons (no potential accidental bitwise AND) and keeps the relevant members from each instance next to one another (making it easier to pick out members that may accidentally be included twice and other relevant mistakes). The above issues don't apply now, but consider if the API changes in future.

Given the include is in a cpp file, it doesn't matter as much.